src/js/components/conversation.jsx

```diff
@@ -136,13 +136,18 @@ export class ConversationComponent extends Component {
             }
         }
         this._lastTopMessageNode = undefined;
-        
+
     };
 
     componentWillUpdate(nextProps) {
         const {messages: currentMessages, isFetchingMoreMessages} = this.props;
         const {messages: newMessages} = nextProps;
 
+        if (!this._lastMessageNode) {
+            this._forceScrollToBottom = true;
```

Comment:
Because scrolling to bottom is the default behavior to have. When this happens, the conversation is empty and is updating to a non-empty state.